src/components/Settings/ChangeSelfPasswordForm.tsx

```diff
@@ -51,8 +51,7 @@ export const ChangeSelfPasswordForm = ({
         actions.resetForm();
       })
       .catch((error) => {
-        // TODO: better form errors
-        actions.setErrors({ old_password: errorToString(error) });
+        actions.setStatus({ old_password: errorToString(error) });
```

Comment:
Probably don't need to set it on old_password specifically.
The error is general, so maybe something like
`setStatus({error: errorToString(error)})` might be better